### ISSUE TITLE
Error in python-jose tutorial...

### ISSUE THREAD

**Issue by raphet** [mpdavis/python-jose#183]
**error:**
`python3.6/site-packages/jose/utils.py", line 87, in base64url_decode     input += b'=' * (4 - rem) TypeError: must be str, not bytes
`

**trigger:**
When one follows your basic tutorial line-by-line...
https://python-jose.readthedocs.io/en/latest/jwk/index.html#verifying-token-signatures

error line: >>> `decoded_sig = base64url_decode(encoded_sig)`

**Comment by blag** [comment]
Good catch. This must have been originally coded under Python 2, where bytes and strings are the same thing. Now with Python 3, that's not the case.

I would be happy to review a pull request fixing this issue. Thanks! 😃

**Comment by david-shiko** [comment]
Still exists

**Comment by blag** [comment]
I would be happy to review and merge a pull request fixing this issue.
